Makefile

```diff
@@ -1026,6 +1029,16 @@ docker-push-manifest-docker-infrastructure: ## Push the multiarch manifest for t
 	$(MAKE) set-manifest-image MANIFEST_IMG=$(CAPD_CONTROLLER_IMG) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="$(CAPD_DIR)/config/default/manager_image_patch.yaml"
 	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="$(CAPD_DIR)/config/default/manager_pull_policy.yaml"
 
+
+.PHONY: docker-push-manifest-test-extension
+docker-push-manifest-test-extension: ## Push the multiarch manifest for the test extension provider images
+	## Minimum docker version 18.06.0 is required for creating and pushing manifest images.
```

Comment:
I can take this as a followup, but I don't think we should include this note as docker 18.06 is EOL for ~4 years at this point.

Reply:
Makes sense. I just copy&pasted it, but let's drop it

Reply:
Done